<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Messaging</class>
 <widget class="QMainWindow" name="Messaging">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;-- Return</string>
    </property>
   </widget>
   <widget class="QLabel" name="firstname">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Messages</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Create New Group</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>400</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Leave Group</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddUserButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>400</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add New User</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>450</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="newMessageEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>261</width>
      <height>75</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="firstname_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Group::Please DoubleClick on a group</string>
    </property>
   </widget>
   <widget class="QListWidget" name="GrouplistWidget">
    <property name="geometry">
     <rect>
      <x>339</x>
      <y>71</y>
      <width>191</width>
      <height>301</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionRectVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="pastMessageView">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>72</y>
      <width>321</width>
      <height>301</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="EnterButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>460</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="userList">
    <property name="geometry">
     <rect>
      <x>535</x>
      <y>71</y>
      <width>201</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>50</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Users in Group</string>
    </property>
   </widget>
   <zorder>EnterButton</zorder>
   <zorder>firstname</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton</zorder>
   <zorder>DeleteButton</zorder>
   <zorder>AddUserButton</zorder>
   <zorder>GrouplistWidget</zorder>
   <zorder>pastMessageView</zorder>
   <zorder>errorBox</zorder>
   <zorder>newMessageEdit</zorder>
   <zorder>firstname_2</zorder>
   <zorder>userList</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>753</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
